postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -336,6 +336,12 @@ def _collect_statement_samples(self):
             tags=self._tags + self._check._get_debug_tags(),
             hostname=self._check.resolved_hostname,
         )
+        self._check.gauge(
```

Comment:
Rather than an internal debug metric, why not just a debug log line?

Reply:
Because metrics are guaranteed to be ingested but logs have to be configured by the customer. And we need to see them over time. We rely on these metrics for troubleshooting the integrations.